.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
Please don't do this. I understand why templates help most people remember what they should include in a good bug report, but disabling this means people like me can't use extensive history as FOSS maintainers to know what we should write and use CLI tools to manage bugs.